lib/modules/versioning/nuget/index.spec.ts

```diff
@@ -1,77 +1,364 @@
 import nuget from '.';
 
 describe('modules/versioning/nuget/index', () => {
-  it.each`
-    input                        | expected
-    ${'9.0.3'}                   | ${true}
-    ${'1.2019.3.22'}             | ${true}
-    ${'3.0.0-beta'}              | ${true}
-    ${'2.0.2-pre20191018090318'} | ${true}
-    ${'1.0.0+c30d7625'}          | ${true}
-    ${'2.3.4-beta+1990ef74'}     | ${true}
-    ${'17.04'}                   | ${true}
-    ${'3.0.0.beta'}              | ${false}
-    ${'5.1.2-+'}                 | ${false}
-  `('isValid("$input") === $expected', ({ input, expected }) => {
-    const res = !!nuget.isValid(input);
-    expect(res).toBe(expected);
+  describe('isSingleVersion()', () => {
+    it.each`
+      input              | expected
+      ${'[1.2.3]'}       | ${true}
+      ${'1.2.3'}         | ${false}
+      ${'[1.2.3,1.2.3]'} | ${false}
```

Comment:
I'm happy for this to return false, but isn't it effectively the same as `[1.2.3]`?

Reply:
Let's stick to the syntactic viewpoint for now. I think this form almost never will be used, unless for some good reason.